index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
@jaumard @LobeTia  Basically we just released version 2. So shouldn't have compatibility issues.
Might be it would be better to create unified interfaces for such things ?
And make all `datastore` trailpacks to use them ? It will solve such issues in future